name.abuchen.portfolio.bootstrap/Application.e4xmi

```diff
@@ -117,7 +117,7 @@
           <visibleWhen xsi:type="ui:ImperativeExpression" xmi:id="_cLemQAxoEe6sSLPjYVyvlw" contributionURI="bundleclass://name.abuchen.portfolio.ui/name.abuchen.portfolio.ui.preferences.DivvyDiaryAPIKeyConfiguredExpression"/>
         </children>
         <children xsi:type="menu:HandledMenuItem" xmi:id="_t1tSoDrlEeydp_NU-UFwGA" elementId="name.abuchen.portfolio.ui.menu.online.syncEODHistoricalData" label="%menu.online.eod-historical-data.label" iconURI="platform:/plugin/name.abuchen.portfolio.ui/icons/eodhistoricaldata-logo.png" command="_hlqdUDrlEeydp_NU-UFwGA">
-          <visibleWhen xsi:type="ui:ImperativeExpression" xmi:id="_t1tSoTrlEeydp_NU-UFwGA" contributionURI="bundleclass://name.abuchen.portfolio.ui/name.abuchen.portfolio.ui.preferences.ExperimentalFeaturesEnabledExpression"/>
+          <visibleWhen xsi:type="ui:ImperativeExpression" xmi:id="_t1tSoTrlEeydp_NU-UFwGA" contributionURI="bundleclass://name.abuchen.portfolio.ui/name.abuchen.portfolio.ui.preferences.EODHistoricalDataAPIKeyConfiguredExpression"/>
```

Comment:
At least for me, that feature does not work with EOD anymore. Something must have been changed.

Actually, I am thinking to remove it - not make it more prominent

Reply:
I don't know if that still works or not. Your judgement call... :-)